setup.py

```diff
@@ -30,6 +30,6 @@
     install_requires=[
         'Django',
         'markdown',
-        'bleach >= 2.0, < 5.0.0',
+        'bleach[css]',
```

Comment:
Should the bleach dependency be set to `>= 5.0.0`?

Reply:
thanks for the hint, I changed it ([909e2a6](https://github.com/erwinmatijsen/django-markdownify/pull/37/commits/909e2a61844cb5945d0af8a3b40dea8210498d9f)).
I also think that it is better with a version specifier.